.changeset/tame-lemons-play.md

```diff
@@ -0,0 +1,170 @@
+---
+"@latticexyz/cli": patch
+"@latticexyz/common": minor
+"@latticexyz/create-mud": minor
```

Comment:
In regular semver, this should _probably_ be `major`? Are we sticking with `minor` for now to keep things on 2.0?